Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixSearchId.tsx

```diff
@@ -219,45 +233,61 @@ export default class EquipmentBigFixSearchId extends React.Component<
   };
 
   private _onSearch = () => {
-    this.setState({ isSearching: true, isFetched: false, isValidSearch: true });
-    this._getComputersBySearch(
+    this.setState({
+      isSearching: true,
+      isFetched: false,
+      isFound: true,
+      isValidSearch: true,
+      listOfComputers: []
+    });
+    this._getComputersBySearchId(
       this.state.selectedField,
       this.state.valueToBeSearched
     );
   };
 
-  private _getComputersBySearch = async (field: string, value: string) => {
+  private _getComputersBySearchId = async (field: string, value: string) => {
     let response = null;
     try {
       response = await this.context.fetch(
-        `/api/${this.context.team.slug}/equipment/GetComputersBySearch?field=${field}&value=${value}`,
-        {
-          method: 'GET'
-        }
+        `/api/${this.context.team.slug}/equipment/GetComputersBySearch?field=${field}&value=${value}`
       );
     } catch (err) {
-      response = null;
-    }
-    // if length is 0, not a valid search
-    if (response.length === 0 || response === null) {
-      this.setState({
-        isFetched: true,
-        isSearching: false,
-        isValidSearch: false
-      });
-    } else {
-      const firstFiveName = response.slice(0, 5);
-      this.setState({
-        isFetched: true,
-        isSearching: false,
-        listOfComputers: firstFiveName
-      });
+      debugger;
```

Comment:
@NabilFurmoli you should take this out, and fix the other tslint warnings on this page. but other than that looks good! :shipit: